src/main/java/com/google/sps/data/GetGovAdaptor.java

```diff
@@ -8,22 +8,24 @@
 import java.io.IOException;
 import java.io.OutputStream;
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 import com.google.gson.Gson;
 import com.google.gson.reflect.TypeToken;
 
 /** File to run the GET request in java before putting it in the main adaptor function. 
  */
 public class GetGovAdaptor {
```

Comment:
Class name `Get...` suggests it would be a Class factory of some kind, that would give you back a `GovAdapter`. This would be better named something like the new `NSWGovAQDataPoint`, possibly `NSWGovAQAdaptor`.